src/middleware/AppContext.tsx

```diff
@@ -0,0 +1,16 @@
+import { createContext, ReactNode, useContext } from 'react';
+import { useFlag } from '@unleash/proxy-client-react';
+
+export interface AppContextInterface {
+  packageVerification: boolean;
+}
+
+export const AppContext = createContext({} as AppContextInterface);
+
+export const ContextProvider = ({ children }: { children: ReactNode }) => {
+  const packageVerification = useFlag('content-sources.package-verification') || false;
```

Comment:
would it make sense to turn this on by default?   what happens in ephemeral (i wouldn't want it to default to false i don't think)?